frontend/src/components/matching/MatchingCard.tsx

```diff
@@ -128,7 +135,7 @@ const MatchingCard = () => {
               </Button>
               <Button
                 className="bg-yellow text-black"
-                onPress={handleGetMatchedButtonPress}
+                onPress={handleGetMatched}
               >
                 Get Matched
```

Comment:
Hi, may I ask what is the difference between the `Get Matched` button and the `Quick Match` button? It seems to me that both of them do the same thing?

Reply:
@tlyi yea ps what was the original intention ah

Reply:
the idea is that Quick Match will start a matching request using the user's preferences while Get Matched takes the preferences in the options to get matched

Reply:
But I think the default options listed in the dashboard already read the user preferences. So, we should just always use te options to start a match. So I will suggest we only leave one button there to prevent confusion.

Reply:
Ah, initially I wanted Quick Match to be easily accessible from the NavBar, so the user can just click on that to start a match with their preferences without having to navigate back to the dashboard. If it is too troublesome to implement, i'm okay with it just being on the dashboard! But if both buttons are in the same card, then I agree that it looks abit redundant. Perhaps we replace it with a small "reset to default" option?